### PR TITLE
Add contribution guidelines and issue-/pr-templates

### PR DESCRIPTION
**Pull request by florianb**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+To make my maintainer's life easier, i...
```
Comment (Arcanemagus) [inline .github/PULL_REQUEST_TEMPLATE.md]: `i` -> `I`

And probably `my` -> `the`?

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# How to contribute
+
+First of all, thank you for stopping by - we would really appreciate if you could support us improving `atom-editorconfig` by giving feedback (:+1: :-1: :memo:), opening new issues :sparkles: or starting pull requests.
+
+If you decide to participate, be kind!
+
+
+### :traffic_light: Roadmap
```
Comment (Arcanemagus) [inline .github/CONTRIBUTING.md]: You skipped a heading level, just for the style difference?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (florianb) [inline .github/PULL_REQUEST_TEMPLATE.md]: I guess we'll leave the small "i", putting "the" in is a good point. :+1:

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (florianb) [inline .github/CONTRIBUTING.md]: Yes.

Comment (slang800) [inline .github/CONTRIBUTING.md]: changing content for style? :cringe:

**Comment by florianb** [comment]
Thanks everybody for support! :clap:

P.s.: Nitpicking refinements are welcome to be done by whoever has time for it. :coffee:

**Comment by sindresorhus** [comment]
Looks very good @florianb :)

**Pull request merged**